.ecp-gitlab-ci.yml

```diff
@@ -33,19 +35,66 @@ stages:
 .Build:
   stage: build
   script:
-    - CI_PROJECT_DIR=${PWD}
     - module load cmake
-    - module load gcc
     - for i in ${BACKENDS}; do CMAKE_OPTS+=( -DCabana_ENABLE_${i}=ON ); done
     - j="$(grep -c processor /proc/cpuinfo 2>/dev/null)" || j=0; ((j++))
     - mkdir build && cd build &&
-      cmake -DCMAKE_PREFIX_PATH=${CI_PROJECT_DIR}/kokkos.install -DCabana_ENABLE_TESTING=ON -DCabana_ENABLE_Serial=OFF -DCabana_ENABLE_EXAMPLES=OFF ${CMAKE_OPTS[@]} .. &&
-      make -k -j${j} VERBOSE=1 &&
-      make test CTEST_OUTPUT_ON_FAILURE=1
+      cmake -DCMAKE_PREFIX_PATH=${CI_PROJECT_DIR}/kokkos.install -DCabana_ENABLE_TESTING=ON -DCabana_ENABLE_Serial=OFF -DCabana_ENABLE_EXAMPLES=ON ${CMAKE_OPTS[@]} ${CMAKE_EXTRA_ARGS} .. &&
+      make -k -j${j} VERBOSE=1
+    - wget https://raw.githubusercontent.com/slashbeast/pax-utils/master/lddtree.sh
```

Comment:
What is this here for?

Reply:
Removed.

Reply:
it is still there...

Reply:
I guess I just wanted to know what the `wget` call did in this case

Reply:
That should be removed! I must have missed that in the squash.

Reply:
Now removed.